files/common/etc/update-motd.d/00-delphix

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+full_version=$(/usr/bin/get-appliance-version)
```

Comment:
why use `/usr/bin` explicitly, vs. relying on using the version in the `PATH`?

Reply:
Just a habit from having had to fix broken scripts that accidentally use incorrect versions of executables because they assumed someone had set PATH correctly for them.

Reply:
IMO, it'd be better to use what's in the PATH; which would also be consistent with how you're calling awk/sed.